crates/medmodels-core/src/medrecord/group_mapping.rs

```diff
@@ -313,15 +313,22 @@ mod test {
         assert_eq!(1, group_mapping.group_count());
 
         group_mapping
-            .add_group("1".into(), Some(vec!["0".into(), "1".into()]), None)
+            .add_group(
```

Comment:
Missing the test on adding a node that does not exist to a group. Is it there already and I have not found it?

Reply:
The Group Mapping itself has no knowledge if a node exists or not. the medrecord does. there is a test for this case in the medrecord :)